content/web-api-rest/studio/_index.md

```diff
@@ -0,0 +1,327 @@
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: 12/15/22 # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+The UI of a browser is designed to make simple `GET` requests for URLs entered into its address bar. This design works great for browsing sites, but falls short when working with APIs. Anything beyond a `GET` request is difficult to send via a browser address bar alone. Think about what is needed to create a new `CodingEvent`. This type of request contains a body. Our MVC application included a view to allow us to test inputs. Our API, however, is **headless**. It does not contain the client-side form. In order to test how it handles requests then, we need a way to interact with the API server without the browser. In this studio, we work with Postman to explore how APIs can be consumed.
+
+## Setup
+
+### Install Postman
+
+If you haven't done so already, [install Postman](https://education.launchcode.org/csharp-web-dev-curriculum/appendices/install-postman/)
+
+### Fork and Clone the API Source Code
+
+With Postman installed, we're ready to fork and clone the [CodingEvents API](https://github.com/LaunchCodeEducation/coding-events-api/). Although it is implemented differently from CodingEvents MVC, you will find that most of the features from the MVC application have been supported through endpoints in the API.
+
+{{% notice blue "Note" "rocket" %}}
+Our focus in this course is on operations and as such we will not be going into the development of the API. However, feel free to explore the source code if you are curious about the similarities and differences between the .NET MVC and API implementations.
+{{% /notice %}}
+
+Let's begin by forking and cloning the repo onto our machine. In your Powershell or terminal window, move into a directory where you plan to save your local copy of the API codebase.
+
+{{% notice blue "Note" "rocket" %}}
+Including a SQLite database in this project means you don't need to have your MySQL server running to test the API. We won't get into what this looks like and instead just concentrate on testing the API endpoints.
+{{% /notice %}}
+
+Let's change into the repo and switch to this branch:
+
+```bash
+# cd is an alias (like a nick-name) for the Set-Location cmdlet in PowerShell
+cd coding-events-api
+```
+
+Keep your terminal open! We have some more work to do.
+
+### Start the API Server
+
+We'll start the API server from the terminal using the `dotnet run` command. Navigate to the `CodingEventsAPI` project folder *within* your `coding-events-api` solution. This is the folder that contains `Controllers/` and so on, and is NOT the main project folder.
+
+```bash
+# change to the CodingEventsAPI project directory
+cd CodingEventsAPI
+```
```

Comment:
```suggestion
```bash
# navigate and change into the coding-events-api solution 
cd coding-events-api
# change to the CodingEventsAPI project directory
cd CodingEventsAPI
```
```